Microsoft.ProviderHub : Update new api-version 2021-05-01-preview

diwudd opened the pull request:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [X] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [ ] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

shenshengkafei committed:
Add operation changes

shenshengkafei committed:
Fix format

shenshengkafei committed:
Fix validations

shenshengkafei committed:
Fix format issue

shenshengkafei committed:
Fix issue

shenshengkafei committed:
Fix issue

erich-wang commented:
@diwudd , is this PR still valid? Please address comment if yes it is valid. We'll close this PR if there's no response from you within 10 biz days, thanks.

diwudd commented:
> @diwudd , is this PR still valid? Please address comment if yes it is valid. We'll close this PR if there's no response from you within 10 biz days, thanks.

Yes, it is valid. I did not see any comment. Yours is the first comment.

erich-wang commented:
> > @diwudd , is this PR still valid? Please address comment if yes it is valid. We'll close this PR if there's no response from you within 10 biz days, thanks.
> 
> Yes, it is valid. I did not see any comment. Yours is the first comment.

Please check the comment from `openapi-workflow-bot`

diwudd commented:
Yes, it is valid. I had addressed the comments.

Thanks,

Di

From: erich-wang ***@***.***>
Sent: Friday, May 28, 2021 2:18 AM
To: Azure/azure-rest-api-specs ***@***.***>
Cc: Di Wu ***@***.***>; Mention ***@***.***>
Subject: Re: [Azure/azure-rest-api-specs] Microsoft.ProviderHub : Update new api-version 2021-05-01-preview (#14438)


@diwudd<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdiwudd&data=04%7C01%7Cwud%40microsoft.com%7C4010f7b069dc41314eb508d921b98c4f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637577903068981638%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=sr0uwHixWuxqoiiIdJd1qtFrCEINHxijyBq03wgDBZ0%3D&reserved=0> , is this PR still valid? Please address comment if yes it is valid. We'll close this PR if there's no response from you within 10 biz days, thanks.

Yes, it is valid. I did not see any comment. Yours is the first comment.

Please check the comment from openapi-workflow-bot

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F14438%23issuecomment-850278418&data=04%7C01%7Cwud%40microsoft.com%7C4010f7b069dc41314eb508d921b98c4f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637577903068981638%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=QpYOMKlQJdunWNqKF0lGV1l%2B%2FUgyXcbDbCICg7W15TE%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAFOVDVR247HT6S5QZYPDV3LTP5NWDANCNFSM45BDEWAQ&data=04%7C01%7Cwud%40microsoft.com%7C4010f7b069dc41314eb508d921b98c4f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637577903068991597%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=PdQ%2Bigp8atlwp7kLDBqh6sVTwlnZ%2BjINwQb%2FSTtJI0s%3D&reserved=0>.

erich-wang commented:
Please follow the comment below:
Hi @diwudd, one or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow [breaking change policy](https://aka.ms/AzBreakingChangesPolicy) to request breaking change review and approval before proceeding swagger PR review. </br>**Action**: To initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).</br> If you want to know the production traffic statistic, please see [ARM Traffic statistic](https://portal.azure-devex-tools.com/api/pr/query-traffic?pr=Azure/azure-rest-api-specs/14438&days=5). </br> If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ms/swaggerfeedback.

erich-wang commented:
@diwudd, is this PR still valid? If yes, please follow the comment below, otherwise we'll close this PR within 10 biz days, thanks.

One or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow [breaking change policy](https://aka.ms/AzBreakingChangesPolicy) to request breaking change review and approval before proceeding swagger PR review. </br>**Action**: To initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).</br> If you want to know the production traffic statistic, please see [ARM Traffic statistic](https://portal.azure-devex-tools.com/api/pr/query-traffic?pr=Azure/azure-rest-api-specs/14438&days=5). </br> If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ms/swaggerfeedback.

diwudd commented:
Hi,

The PR is valid.

The breaking changes are only introduced in the new api-version.

The breaking changes are required as the resource schema was not following ARM resource schema and we have to make it use ARM resource schema so that customers can create the resources via ARM template deployment.

Thanks,

Di

From: erich-wang ***@***.***>
Sent: Monday, June 7, 2021 1:46 AM
To: Azure/azure-rest-api-specs ***@***.***>
Cc: Di Wu ***@***.***>; Mention ***@***.***>
Subject: Re: [Azure/azure-rest-api-specs] Microsoft.ProviderHub : Update new api-version 2021-05-01-preview (#14438)


Please follow the comment below:
Hi @diwudd<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdiwudd&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624591671%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=eUI0bYd6F%2FAaff%2FMGTz1p3PpwHcZl8c4XArBzCWvbBg%3D&reserved=0>, one or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow breaking change policy<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faka.ms%2FAzBreakingChangesPolicy&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624601632%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=X50ule0%2BY818EIZPlcBx4RurzDp0YXz7waoDQ1uQP2c%3D&reserved=0> to request breaking change review and approval before proceeding swagger PR review.
Action: To initiate an evaluation of the breaking change, create a new intake using the template for breaking changes<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faka.ms%2FBreakingchangetemplate&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624601632%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=HnpNnU6gB0fM%2FZXUOD3AIFdUhiV4d6aJBBwxIXOQwEc%3D&reserved=0>. Addition details on the process and office hours are on the Breaking change Wiki<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdev.azure.com%2Fmsazure%2FAzureWiki%2F_wiki%2Fwikis%2FAzureWiki.wiki%2F37684%2FBreaking-Changes&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624601632%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Bc1Iaf0BnYKmZGFat2Oh7Rl9G54GB%2FxDa4StarRZ6mI%3D&reserved=0>.
If you want to know the production traffic statistic, please see ARM Traffic statistic<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fportal.azure-devex-tools.com%2Fapi%2Fpr%2Fquery-traffic%3Fpr%3DAzure%2Fazure-rest-api-specs%2F14438%26days%3D5&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624611583%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=d0YwRjIXSnEvr0KVOAiUb%2BpXTw2ociP%2Ftg091Y%2BeMGc%3D&reserved=0>.
If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ms/swaggerfeedback<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faka.ms%2Fswaggerfeedback&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624611583%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ztMnzrIuH7N%2BFaWzHjf10eV4lgAOmdZ8vtnrq6wyBts%3D&reserved=0>.

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F14438%23issuecomment-855735505&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624621540%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Gi%2BP9CeXx1RyyqszhxoDI1hKvLqIQlc4izTj55Txaf0%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAFOVDVQCPM63HT755NAPSNTTRSBMPANCNFSM45BDEWAQ&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624621540%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=0PFD2E5p%2BY6194uY%2BgfMFkueQ8yIY%2FaYhdtfVLs3kyo%3D&reserved=0>.

diwudd commented:
Hi,

Can you please take a look at the PR and let me know if you have any questions?

Thanks,

Di

From: Di Wu
Sent: Wednesday, June 23, 2021 10:13 AM
To: Azure/azure-rest-api-specs ***@***.***>; Azure/azure-rest-api-specs ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [Azure/azure-rest-api-specs] Microsoft.ProviderHub : Update new api-version 2021-05-01-preview (#14438)

Hi,

The PR is valid.

The breaking changes are only introduced in the new api-version.

The breaking changes are required as the resource schema was not following ARM resource schema and we have to make it use ARM resource schema so that customers can create the resources via ARM template deployment.

Thanks,

Di

From: erich-wang ***@***.******@***.***>>
Sent: Monday, June 7, 2021 1:46 AM
To: Azure/azure-rest-api-specs ***@***.******@***.***>>
Cc: Di Wu ***@***.******@***.***>>; Mention ***@***.******@***.***>>
Subject: Re: [Azure/azure-rest-api-specs] Microsoft.ProviderHub : Update new api-version 2021-05-01-preview (#14438)


Please follow the comment below:
Hi @diwudd<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdiwudd&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624591671%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=eUI0bYd6F%2FAaff%2FMGTz1p3PpwHcZl8c4XArBzCWvbBg%3D&reserved=0>, one or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow breaking change policy<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faka.ms%2FAzBreakingChangesPolicy&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624601632%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=X50ule0%2BY818EIZPlcBx4RurzDp0YXz7waoDQ1uQP2c%3D&reserved=0> to request breaking change review and approval before proceeding swagger PR review.
Action: To initiate an evaluation of the breaking change, create a new intake using the template for breaking changes<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faka.ms%2FBreakingchangetemplate&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624601632%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=HnpNnU6gB0fM%2FZXUOD3AIFdUhiV4d6aJBBwxIXOQwEc%3D&reserved=0>. Addition details on the process and office hours are on the Breaking change Wiki<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdev.azure.com%2Fmsazure%2FAzureWiki%2F_wiki%2Fwikis%2FAzureWiki.wiki%2F37684%2FBreaking-Changes&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624601632%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Bc1Iaf0BnYKmZGFat2Oh7Rl9G54GB%2FxDa4StarRZ6mI%3D&reserved=0>.
If you want to know the production traffic statistic, please see ARM Traffic statistic<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fportal.azure-devex-tools.com%2Fapi%2Fpr%2Fquery-traffic%3Fpr%3DAzure%2Fazure-rest-api-specs%2F14438%26days%3D5&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624611583%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=d0YwRjIXSnEvr0KVOAiUb%2BpXTw2ociP%2Ftg091Y%2BeMGc%3D&reserved=0>.
If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ms/swaggerfeedback<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faka.ms%2Fswaggerfeedback&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624611583%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ztMnzrIuH7N%2BFaWzHjf10eV4lgAOmdZ8vtnrq6wyBts%3D&reserved=0>.

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-rest-api-specs%2Fpull%2F14438%23issuecomment-855735505&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624621540%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Gi%2BP9CeXx1RyyqszhxoDI1hKvLqIQlc4izTj55Txaf0%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAFOVDVQCPM63HT755NAPSNTTRSBMPANCNFSM45BDEWAQ&data=04%7C01%7Cwud%40microsoft.com%7C1c53ab21b80a48603a2c08d92990ace3%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637586523624621540%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=0PFD2E5p%2BY6194uY%2BgfMFkueQ8yIY%2FaYhdtfVLs3kyo%3D&reserved=0>.

erich-wang commented:
There is dedicated team instead of me to review the breaking change, so please follow process:

One or multiple breaking change(s) is detected in your PR. Please check out the breaking change(s), and provide business justification in the PR comment and @ PR assignee why you must have these change(s), and how external customer impact can be mitigated. Please ensure to follow [breaking change policy](https://aka.ms/AzBreakingChangesPolicy) to request breaking change review and approval before proceeding swagger PR review. </br>**Action**: To initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).</br> If you want to know the production traffic statistic, please see [ARM Traffic statistic](https://portal.azure-devex-tools.com/api/pr/query-traffic?pr=Azure/azure-rest-api-specs/14438&days=5). </br> If you think it is false positive breaking change, please provide the reasons in the PR comment, report to Swagger Tooling Team via https://aka.ms/swaggerfeedback.
